config/initializers/sufia_catalog_search_builder_overrides.rb

```diff
@@ -18,4 +18,9 @@
     klass.send(:include, SearchBuilder::RestrictAdminSearchFields)
   end
 
+  unless klass.ancestors.include? SearchBuilder::SyntheticCategoryLimit
+    klass.send(:include, SearchBuilder::SyntheticCategoryLimit)
+    klass.synthetic_category_param = :temp_synthetic_collection
```

Comment:
I don't see this in the include above in app/models/search_builder. So I think that means we only want this behavior in the catalog search builder. Then I'm not sure why the module is included in app/models/search_builder at all.